src/components/pack/NsfwModal.tsx

```diff
@@ -0,0 +1,74 @@
+import {styled} from 'linaria/react';
```

Comment:
Noticed mixed indentation between 2 spaces and 4 spaces in this file. We use 2 spaces in the rest of the repo. I've opened https://github.com/signalstickers/signalstickers/pull/152 which adds an `.editorconfig` file to the repository which should help to automatically prevent these issues in the future.